src/transforms/coalesce.jl

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+struct Coalesce{T} <: Colwise
```

Comment:
Missing docstring

Reply:
Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)